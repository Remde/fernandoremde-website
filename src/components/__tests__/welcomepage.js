import React from "react"
import renderer from "react-test-renderer"

import Welcome from "../welcomePage.js"

describe("Welcome", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Welcome />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
