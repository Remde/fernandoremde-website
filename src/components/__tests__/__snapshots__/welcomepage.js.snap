// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome renders correctly 1`] = `
<div
  className="welcomeSection"
>
  <div
    className="welcomeText"
  >
    <h1
      className="mainText"
    >
      fernando remde
    </h1>
    <p
      className="subText"
    >
      a portfolio website. I also write 
      <a
        href="/blog"
      >
        here
      </a>
      .
    </p>
  </div>
</div>
`;
